tests/unit/acquisition/test_optimizer.py

```diff
@@ -824,3 +826,64 @@ def side_effect(*args: Any, **kwargs: Any) -> spo.OptimizeResult:
     else:
         received_constraints = None
     assert received_constraints == expected_constraints
+
+
+@pytest.mark.parametrize("num_initial_points", [0, 1, 2, 3, 4])
+def test_generate_initial_points(num_initial_points: int) -> None:
+    def sampler(space: SearchSpace) -> Iterable[TensorType]:
+        assert space == Box([-1], [2])
```

Comment:
you are using this space in multiple tests, perhaps create a constant at the top and reuse it? or a fixture?